Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -248,5 +252,32 @@ public async Task PostAsyncComment_InvalidPost_BadRequest()
             Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
         }
 
+
+
+        [Fact]
+        public void GetNewsFeedAsyn_DtoIsNotNull_ReturnsOk()
```

Comment:
typo in name. also, this is a unit test, not an integration test. why is it in this project?

Reply:
fixed typo in name